doc/manual/en/index.rst

```diff
@@ -8,23 +8,78 @@ It can access the KNX-Bus directly (via eibd/knxd) or use
 `OpenHAB <http://www.openhab.org/>`__ as a backend. The current version
 can be found on `Github <http://github.com/CometVisu/CometVisu>`__.
 
-.. toctree::
-    :hidden:
-    :glob:
+.. _system-requirements:
 
-    install/index
+System Requirements
+-------------------
+
+Operation by the user takes place via a web browser. The
+Most current browsers are supported.
```

Comment:
`The Most current browsers` -> `Most of the current browsers`
(Die alleraktuellsten Browser -> Die meisten der aktuellen Browser)